concent_api/verifier/utils.py

```diff
@@ -36,7 +36,7 @@
 from .constants import UNPACK_CHUNK_SIZE
 
 
-sys.path.append('/golem/entrypoints/scripts/render_tools')
+
```

Comment:
Why is this change visible as removal (instead of just disappearing? Is this already in master?

Reply:
This change is visible because it was added in previous PR. In this PR I am changing it to work with custom directory